scrunch/datasets.py

```diff
@@ -1025,8 +971,17 @@ def itervalues(self):
             yield Variable(var_tuple, self)
 
     def iterkeys(self):
+        """
+        Yield variable alias, since they are unique
+        """
         for var in self._vars:
```

Comment:
If you have the `self._catalog` you could better do `return self._catalog.by('alias).iterkeys()`